Sasquatch/Sasquatch.xcodeproj/project.pbxproj

```diff
@@ -23,8 +23,6 @@
 		84226C561E88FF4A00798417 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 84226C551E88FF4A00798417 /* Assets.xcassets */; };
 		84226C831E8908CA00798417 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 84226C821E8908CA00798417 /* AppDelegate.swift */; };
 		84226C8A1E8908CA00798417 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 84226C891E8908CA00798417 /* Assets.xcassets */; };
-		84226D0D1E891FEF00798417 /* MSAlertController.m in Sources */ = {isa = PBXBuildFile; fileRef = 84226D0B1E891FBA00798417 /* MSAlertController.m */; };
```

Comment:
This would be an issue if Distribute is not included in the project. Distribute is the only one contains MSAlertController symbols, it should stay here. As your concern is -ObjC flag on customer's application but this is a test application provided by us, any changes made in this test application by others are not guaranteed to be working.
Keep this file here or we need consider to use UIAlertController instead of MSAlertController.